### PR TITLE
Limit nominate command channels

### PR DESCRIPTION
**Pull request by wookie184**
This limits the `!nominate` command to the nominations channel, adds a force nominate command so mods can override that, and also adds the `moderators` category to the config as a moderation category, so `nomination-discussion` and any new channels added in the future can have moderation commands run in them.

### CODE REVIEW

File: config-default.yml [hunk 1]
```diff
@@ -193,12 +194,10 @@ guild:
         helpers:            &HELPERS        385474242440986624
         incidents:                          714214212200562749
         incidents_archive:                  720668923636351037
-        mods:               &MODS           305126844661760000
+        mods:                               305126844661760000
         mod_alerts:                         473092532147060736
-        mod_appeals:        &MOD_APPEALS    808790025688711198
```
Comment (Akarys42) [inline config-default.yml:198]: Since those channels have been from here, they also need to be removed from `bot/constants.py`.

**Review by Akarys42 — CHANGES_REQUESTED** [review]
Just one small thing!

**Comment by wookie184** [comment]
I don't think the ones I removed were in the constants.py (unless I'm missing something)

**Comment by MarkKoz** [comment]
Why make a separate command instead of allowing the original one to work everywhere for moderators?

**Comment by wookie184** [comment]
> 
> 
> Why make a separate command instead of allowing the original one to work everywhere for moderators?

To discourage moderators from running the nominate command in channels other than the `nominations` channel, for example if they clicked on the wrong channel or didn't realise they should only use the `nominations` channel, while still allowing them to bypass that if necessary. Granted theses cases would be rare, but I don't think there's much harm in adding a separate command.

**Review by Akarys42 — APPROVED** [review]

**Review by Akarys42 — APPROVED** [review]
you can do it review bot

**Review by Xithrius — APPROVED** [review]
Looks good to my eyes

**Review by Akarys42 — APPROVED** [review]
I don't like you review bot

**Pull request merged**